libs/community/langchain_community/llms/ipex_llm.py

```diff
@@ -10,22 +10,22 @@
 logger = logging.getLogger(__name__)
 
 
-class BigdlLLM(LLM):
-    """Wrapper around the BigDL-LLM Transformer-INT4 model
+class IPEXLLM(LLM):
+    """Wrapper around the IPEX-LLM model
 
     Example:
         .. code-block:: python
 
-            from langchain.llms import TransformersLLM
-            llm = TransformersLLM.from_model_id(model_id="THUDM/chatglm-6b")
+            from langchain.llms import IPEXLLM
```

Comment:
from langchain_community.llms import IPEXLLM

Reply:
also you'll need to update `langchain_community/llms/__init__.py` for this to work